### ISSUE TITLE
Add flow types to AccessibilityAndroidExample.android.js

### ISSUE THREAD

**Issue by kacieb** [facebook/react-native#31372]
## Description
[AccessibilityAndroidExample.android.js](https://github.com/facebook/react-native/blob/master/packages/rn-tester/js/examples/Accessibility/AccessibilityAndroidExample.android.js) does not have proper Flow types. Without Flow typing enforced, it is easy for bugs to be introduced when making changes to this file. This issue details the work to enforce Flow typing in this file.

Steps:
1. Add the `@flow` annotation to AccessibilityAndroidExample.android.js
2. Run `flow status` and fix any Flow errors that appear.
    * You can check out Flow's getting started guide here: https://flow.org/en/docs/getting-started/
    * You can check out Flow's React-specific documentation here: https://flow.org/en/docs/react/components/
    * An example file that is already annotated with `@flow`: [ScrollViewExample.js](https://github.com/facebook/react-native/blob/master/packages/rn-tester/js/examples/ScrollView/ScrollViewExample.js)
3. **Bonus:** Update the file to use [`@flow strict-local`](https://flow.org/en/docs/strict/#toc-strict-local)

## Expected Results
AccessibilityAndroidExample.android.js is annotated with proper flow types.

**Comment by sidverma32** [comment]
@kacieb  I would like to take this up. As this will be my first contribution to this repo. if you can guide me it will be very helpful.

**Comment by kacieb** [comment]
> @kacieb I would like to take this up. As this will be my first contribution to this repo. if you can guide me it will be very helpful.

Awesome! Thanks so much for taking this on! The [react-native repo's readme](https://github.com/facebook/react-native) is a good place to start. You can also check out the [contribution guide](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md).

Feel free to comment on this issue if you run into any unusual Flow errors and need help!

Once you're done, you can submit a PR and tag me as a reviewer! Thanks again for working on this :)

**Comment by sidverma32** [comment]
When I am trying to run the command `"flow": "flow"` from `package.json`, what I am seeing that every file has some `Type annotations` problem!
Is that expected or anything i am missing here? @kacieb

**Comment by kacieb** [comment]
Nope that's definitely unexpected! Looks like something isn't set up properly for Flow, it seems to think you're using Typescript. Google might be able to help, or the Flow documentation.

**Comment by sidverma32** [comment]
Hi @kacieb ,
There is some unusual behaviour i am seeing in project with `flow@0.149.0`  that filename like this `AccessibilityAndroidExample.android.js` are having "zero coverage" but if you change the extension to `.js ` only the coverage goes to 100% with same codebase.

Any comment on this?

Preview of flow coverage having extensions  this `.android.js`
<img width="177" alt="Screenshot 2021-04-28 at 10 36 43 PM" src="https://user-images.githubusercontent.com/10995431/116444281-368ed300-a872-11eb-8327-461cb0414942.png">
Preview of flow coverage having extensions  this `.js`
<img width="177" alt="Screenshot 2021-04-28 at 10 39 07 PM" src="https://user-images.githubusercontent.com/10995431/116444642-8c637b00-a872-11eb-8f12-408524449d64.png">

**Comment by kacieb** [comment]
I think you can run `flow check --flowconfig-name .flowconfig.android` to check Android! That seems to be what's described [here](https://github.com/facebook/react-native/blob/master/package.json#L61).

I don't have a ton of context on why `.android.js` files are ignored, but let me know if this solution helps you. Thanks for bringing up this issue, I'll make sure any other issues with `.android.js` files mention this!

**Comment by linco95** [comment]
> When I am trying to run the command `"flow": "flow"` from `package.json`, what I am seeing that every file has some `Type annotations` problem!
> Is that normal or anything i am missing here? @kacieb
> 
> <img alt="Screenshot 2021-04-22 at 9 19 06 PM" width="1092" src="https://user-images.githubusercontent.com/10995431/115744639-617bb180-a3b0-11eb-81a6-8e2e2c2d7bee.png">

Try installing the [flow language support](https://marketplace.visualstudio.com/items?itemName=flowtype.flow-for-vscode) vs code extension if you haven't already. I was also pretty confused by these errors the first time I saw them, until I realised it was a flow project.

**Comment by sidverma32** [comment]
> > When I am trying to run the command `"flow": "flow"` from `package.json`, what I am seeing that every file has some `Type annotations` problem!
> > Is that normal or anything i am missing here? @kacieb
> 
> Try installing the [flow language support](https://marketplace.visualstudio.com/items?itemName=flowtype.flow-for-vscode) vs code extension if you haven't already.

It's already enabled in my VScode @linco95 and did all the config. which was there in [flow language support]

(https://marketplace.visualstudio.com/items?itemName=flowtype.flow-for-vscode) . But unfortunately , no luck! 😒
<img width="876" alt="Screenshot 2021-05-01 at 7 02 02 PM" src="https://user-images.githubusercontent.com/10995431/116784063-b81f7480-aaaf-11eb-86a4-a2aa92b9a64d.png">



Anyway, using the above command which @kacieb  suggested, with that i was able to debug and resolve my `flow` errors in file.(but not able to track the coverage in VScode)

@kacieb  I have raised the PR #31428 , can you please have a look over it and let me know if any modification required?

**Comment by sidverma32** [comment]
Thanks @kacieb for your help and support in closing this issue 🙏🏻🎉 . It was really great working experience on this PR and exploring things inside `react-native` 😊😎 [ Looking forward for some more challenges 😉]

**Comment by kacieb** [comment]
@sidverma32 feel free to keep an eye on the [React Native project boards](https://github.com/facebook/react-native/projects) if you are interested to keep contributing! We're still working on adding to the Code Quality project, and the Accessibility project has a bunch!

Thanks so much for tackling this issue and making React Native better! 🎉

**Comment by sidverma32** [comment]
Sure @kacieb , i will keep an eye on it. Just wanted to checkout with you is there any common channel/platform/discussion-forum where outside people can connect with the FB engineers or post/discuss problems in a group or get updates on [engineering channel] what/how core react-native team is solving engineering problems [ eg: we use Slack :P ]
